### ISSUE TITLE
Container images issue

### ISSUE THREAD

**Issue by 313mlclub** [quarkusio/quarkus-workshops#237]
I am running a test according to the contents of the following link:  
https://quarkus.io/quarkus-workshops/super-heroes/index-azure.html#_running_remote_containers

But container image links mentioned in the file "quarkus-workshop-super-heroes/super-heroes/infrastructure/docker-compose-app-remote.yaml" cannot be accessed. 
ACR path (registrysuperheroesagoncal.azurecr.io) is not available.

**Comment by cescoffier** [comment]
That's a question for @agoncal

**Comment by agoncal** [comment]
@313mlclub have you built your own images and pushed them to the Azure Container Registry?

**Comment by 313mlclub** [comment]
> @313mlclub have you built your own images and pushed them to the Azure Container Registry?

Yes, I have. But I was not able to build my own images. There were several issues when I tried to run "docker build" commands. Therefore, if you have time, could you push the finished images to Docker Hub or another public registry so that I can test the Azure environment with the already created image?

**Comment by holly-cummins** [comment]
I wonder if we should include an image-building step in the CI?
